components/common/resources/ome/config.xml

```diff
@@ -249,6 +249,11 @@
        <property name="mutable" value="false"/>
        <property name="visibility" value="all"/>
     </bean>
+    <bean class="ome.system.Preference" id="omero.webgateway.max_tile_length">
```

Comment:
name it ``omero.pixeldata.max_tile_length`` to be in synch with the rest
It might not only be used in webgateway